src/mockdata.jsx

```diff
@@ -0,0 +1,145 @@
+const ludos = [
+    {
+        "id": "472",
+        "title": "The Elder Scrolls V: Skyrim",
+        "imagesrc": "/game/472.png",
+        "year": "2011",
```

Comment:
Change this into a number. Same as everywhere else where there's a string of a number.

Reply:
Mock data's numbers has been fixed, Image addresses has been added to the mock data.